src/views/log.blade.php

```diff
@@ -68,13 +68,13 @@
       <div class="list-group">
         @foreach($folders as $folder)
           <div class="list-group-item">
-            <a href="?f={{ \Illuminate\Support\Facades\Crypt::encrypt($folder) }}">
+            <a href="?f={{ encrypt($folder) }}">
```

Comment:
Since this lib is compatible with Laravel _and_ Lumen, you can't just use the `encrypt` helper, you have (sorry about that) to use the full `\Illuminate\Support\Facades\Crypt::encrypt` way :/